www/js/services.js

```diff
@@ -102,6 +102,91 @@ angular.module('emission.services', [])
                "Email cancel reported, seems to be an error on android");
         });
     }
+
+    this.dataCollectionSetConfig = function(config) {
+      return new Promise(function(resolve, reject) {
+          resolve(window.cordova.plugins.BEMDataCollection.setConfig(config));
+      });
+    };
+
+    this.dataCollectionGetConfig = function() {
+      return new Promise(function(reslove, reject) {
+          reslove(window.cordova.plugins.BEMDataCollection.getConfig());
+      });
+    };
+    this.serverSyncSetConfig = function(config) {
+      return new Promise(function(resolve, reject) {
```

Comment:
these are not yet promisified, but since they are our plugins, we should just change our plugin to be promisified.

You just make the change to this repo:
https://github.com/e-mission/cordova-server-sync